### PR TITLE
DOCS-8089 API groups

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-8089 API groups

### CODE REVIEW

File: modules/ROOT/pages/publish-apis.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-As an Anypoint API Community Manager community administrator, you can add and publish APIs and minor and major updates from Anypoint Exchange into your community. 
+As an Anypoint API Community Manager community administrator, you can add and publish APIs and API Groups and minor and major updates from Anypoint Exchange into your community.
```
Comment (priyaayyar) [inline modules/ROOT/pages/publish-apis.adoc]: As an Anypoint API Community Manager community administrator, you can add and publish APIs and API Groups and minor and major updates from Anypoint Exchange into your community. 
I'm thinking this is the first reference to ACM that is why you are going the whole nine yards. If not, you can do ACM right?
As an Anypoint API Community Manager community administrator, you can publish (updated API Groups or) updates API Groups from Anypoint Exchange into your community?

File: modules/ROOT/pages/publish-apis.adoc [hunk 2]
```diff
@@ -126,7 +126,7 @@ Consumer developers can no longer see the previous API version in the API carous
 
 If no changes are visible, xref:troubleshooting.adoc#refresh-the-cache[refresh the cache].
 
-== Searching APIs
+== Searching APIs and API Groups
```
Comment (priyaayyar) [inline modules/ROOT/pages/publish-apis.adoc]: Searching for APIs and API Groups?

**Review by priyaayyar — APPROVED** [review]

File: modules/ROOT/pages/publish-apis.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/publish-apis.adoc]: Correct, this is the first reference to ACM.

Updated the text for clarity.

File: modules/ROOT/pages/publish-apis.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/publish-apis.adoc]: That is better. Fixed.

**Pull request merged**